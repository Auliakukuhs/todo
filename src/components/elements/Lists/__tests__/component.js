import React from 'react';
import { shallow } from 'enzyme';
import Component from '../component';

jest.mock('@material-ui/core/Lists');

describe('Lists component', () => {
  let props = {};

  beforeEach(() => {
    props = {
      classes: {}
    };
  });

  it('render correctly', () => {
    const wrapper = shallow(<Component {...props} />);
    expect(wrapper.exists()).toMatchSnapshot();
  });
});
