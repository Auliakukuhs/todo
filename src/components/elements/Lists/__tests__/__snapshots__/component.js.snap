// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu component render correctly 1`] = `true`;
